components/mosart/src/riverroute/RtmMod.F90

```diff
@@ -1595,6 +1595,10 @@ subroutine Rtmini(rtm_active,flood_active)
        TRunoff%wh   = rtmCTL%wh
        TRunoff%wt   = rtmCTL%wt
        TRunoff%wr   = rtmCTL%wr
+       TRunoff%mr   = rtmCTL%mr
+       TRunoff%yr   = rtmCTL%yr
+       TRunoff%pr   = rtmCTL%pr
+       TRunoff%rr   = rtmCTL%rr
```

Comment:
It is not quite clear to me why we need these channel cross-section geometry values. In principle, they can all be derived from the channel storage by invoking the updatestate_mainchannel subroutine.